### PR TITLE
Adding Fedora 23 requirements to README

### PR DESCRIPTION
**Pull request by antonym**

### CODE REVIEW

**Comment by KennethWilke** [comment]
:+1:

**Comment by csmart** [comment]
@jenkins-openbmc ok to test

File: README.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ OpenBMC uses Yocto/Open-Embedded for a build system, which supports an
 out-of-tree build.  It is recommended that you create an empty directory
 somewhere to hold the build.  This directory will get big.
 
+On Fedora 23 the following packages are required to build the default target:
+
+    dnf install -y git patch diffstat texinfo chrpath SDL-devel bitbake
```
Comment (csmart) [inline README.md]: I'm not sure that we will need bitbake from Fedora, as we use the built-in bitbake at https://github.com/openbmc/openbmc/blob/master/yocto-poky/bitbake/bin/bitbake

Have you tried to build on Fedora without bitbake installed?

Also, this commit will probably conflict with #121 so we should perhaps look to merge one first and then fix up this one.

**Comment by KennethWilke** [comment]
Since bitbake is provided as a submodule of this repo it probably isn't required. I don't mind changing my PR if you want to merge this one first, but having documentation that describes what is needed to build the project is pretty handy.

**Comment by antonym** [comment]
Yeah, it wouldn't really do much until I installed bitbake from the repo so it seemed like it needed to be installed at least.  I can drop it if you don't think it's needed.  If you want to merge #121 first, I'll fix up the conflict on this one.

**Comment by williamspatrick** [comment]
@antonym - Ready for rebase and update.

**Comment by antonym** [comment]
Rebased and should be ready.

File: README.md [hunk 2]
```diff
@@ -4,19 +4,21 @@
...
 ## Building ##
 
-OpenBMC uses Yocto/Open-Embedded for a build system, which supports an 
+OpenBMC uses Yocto/Open-Embedded for a build system, which supports an
 out-of-tree build.  It is recommended that you create an empty directory
 somewhere to hold the build.  This directory will get big.
 
 On Ubuntu 14.04 the following packages are required to build the default target
-```
-sudo apt-get install -y git build-essential libsdl1.2-dev texinfo gawk chrpath diffstat
```
Comment (csmart) [inline README.md]: Looks like we've lost the code tags around these shell commands, are you able to add those back and perhaps add them to your fedora commands also?

**Comment by antonym** [comment]
Should be all cleaned up and ready to go now.

**Pull request merged**